convert2rhel/actions/pre_ponr_changes/kernel_modules.py

```diff
@@ -67,8 +76,26 @@ def _get_rhel_supported_kmods(self):
         if system_info.version.major == 8:
             basecmd.append("--setopt=module_platform_id=platform:el8")
```

Comment:
<nod>  Do you think we could simply add the setopt to the yum makecache and it would be enough to mitigate any potential problems or do we need to figure out what problems could exist and whether doing that is safe and effective? 
```suggestion
            basecmd.append("--setopt=module_platform_id=platform:el8")
            precache.append("--setopt=module_platform_id=platform:el8")            
```

Reply:
I think simply setting it is fine. Thanks.